src/main/java/org/cryptomator/common/settings/InvalidSetting.java

```diff
@@ -0,0 +1,43 @@
+package org.cryptomator.common.settings;
+
+import java.net.URI;
+import java.util.Objects;
+import java.util.ResourceBundle;
+
+public enum InvalidSetting implements AbstractInvalidSetting {
```

Comment:
What is the purpose of this interface, if there is just one implementing class that is publicly accessible as well?

Reply:
Fixed by 714a0c6664cd4091fd607b85fb2153d4a82cebee